mm2src/coins/eth.rs

```diff
@@ -6457,8 +6478,8 @@ pub async fn eth_coin_from_conf_and_request(
         return ERR!("Failed to get client version for all urls");
     }
 
-    let (coin_type, decimals) = match protocol {
-        CoinProtocol::ETH { .. } => (EthCoinType::Eth, ETH_DECIMALS),
+    let (coin_type, decimals, chain_id) = match protocol {
+        CoinProtocol::ETH { chain_id } => (EthCoinType::Eth, ETH_DECIMALS, chain_id),
```

Comment:
ummm, missed that.
does this mean this was never backward compatible to begin with (since we are extracting protocol info, whose format has changed).

Reply:
Yep, we needed to add `chain_id` in it's new place but keep the old one as well. Now we don't need the old one.